frontend/package.json

```diff
@@ -13,10 +13,11 @@
     "enzyme-adapter-react-16": "^1.15.5",
     "jest-sonar-reporter": "^1.3.0",
     "prop-types": "^15.7.2",
-    "react": "^16.13.1",
+    "react": "^16.14.0",
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.3"
+    "react-scripts": "4.0.0",
+    "typescript": "^4.0.5"
```

Comment:
why did we have to add typescript?

Reply:
It's a dependency to one of our dependencies

Reply:
No you can keep that